COLLABORATOR_GUIDE.md

```diff
@@ -73,6 +73,70 @@ All pull requests that modify executable code should be subjected to
 continuous integration tests on the
 [project CI server](https://ci.nodejs.org/).
 
+## Semver-major Changes
+
+Changes to API arguments, return values, error handling, timing, and
```

Comment:
I would stick with a general definition of a breaking change. I don't think the description here is accurate. For example, adding an API argument could be `semver-minor` if it does not change behavior in code that does not use the new argument.

If you do want to include a list of examples of the types of things that are breaking changes, I would put that after a general definition rather than coming out of the gate with it.